blazor/combobox/style.md

```diff
@@ -136,6 +136,34 @@ Use the following CSS to customize the appearance of popup element.
 }
 ```
 
+## Readonly-mode
+
+Specify the boolean value to [Readonly](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Readonly), whether the DropDownList allows the user to change the value.
+
+If you are setting Readonly property, then you could not interact the component. You can set initial value to the component alone.  
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/readonly-mode.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Readonly mode](./images/style/blazor_combobox_readonly-mode.png)
+
+## Show clear Button
+
+Use the [ShowClearButton](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfComboBox-2.html#Syncfusion_Blazor_DropDowns_SfComboBox_2_ShowClearButton) property to specify whether to show or hide the clear button. When the clear button is clicked, `Value`, `Text`, and `Index` properties are reset to null.
```

Comment:
Use the [ShowClearButton](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfComboBox-2.html#Syncfusion_Blazor_DropDowns_SfComboBox_2_ShowClearButton) property to specify whether to show or hide the clear button. When the clear button is clicked, the `Value,` `Text,` and `Index` properties are reset to null.